ami/flowchart/library/Editors.py

```diff
@@ -118,7 +139,7 @@ def restoreState(self, state):
 class HistEditor(TraceEditor):
 
     def __init__(self, parent=None, **kwargs):
-        kwargs['uiTemplate'] = [('brush', 'color', {'value': kwargs.get('color', (255, 0, 0))})]
+        kwargs['uiTemplate'] = [('Brush', 'color', {'value': kwargs.get('color', (255, 0, 0))})]
```

Comment:
This was a typo. In order to make the stylesheet consistent I changed it to Brush from brush. This means existing plots that customized the histogram color will lose the color unless you edit the fc file and change it from brush to Brush, or you could just reset the color in the editor manually.